rawpointer_test.go

```diff
@@ -232,3 +232,24 @@ func TestRawPointerTotalListSize(t *testing.T) {
 		}
 	}
 }
+
+func TestLandingPadNearPointer(t *testing.T) {
+	tests := []struct {
+		far     rawPointer
+		tag     rawPointer
+		landing rawPointer
+	}{
+		{0x08, 0x2000200000000, 0x2000200000000},
```

Comment:
These tests would also benefit in being rewritten in terms of rawPointer constructors.  For example, this first one would be:

```go
{rawFarPointer(0, 8), rawStructPointer(0, ObjectSize{16, 2}), rawStructPointer(0, ObjectSize{16, 2})},
```